server/src/test/java/org/elasticsearch/action/admin/indices/close/CloseIndexResponseTests.java

```diff
@@ -113,7 +113,11 @@ protected void assertEqualInstances(CloseIndexResponse expected, CloseIndexRespo
                             assertThat(actualFailure.getNodeId(), equalTo(expectedFailure.getNodeId()));
                             assertThat(actualFailure.index(), equalTo(expectedFailure.index()));
                             assertThat(actualFailure.shardId(), equalTo(expectedFailure.shardId()));
-                            assertThat(actualFailure.reason(), equalTo(expectedFailure.reason()));
+                            // Serialising and deserialising an exception seems to remove the "java.base/" part from the stack trace,
+                            // so these string replacements account for this.
+                            assertThat(
+                                actualFailure.reason().replace("java.base/", ""),
```

Comment:
Can we check for a portion of the failure instead of an exact stack trace?

Reply:
Yes, I've reworked both instances of this.